speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/CHANGES.txt

```diff
@@ -46,12 +48,17 @@ Added:
             ResearcherMenu.xml
             DataSelectionMenu.xml
             DeveloperMenu.xml
+            EndExperimentMenu.xml
+            raceresultsmenu.xml
```

Comment:
This is the list of files we made, below is the list of files we from SD where we changed something